spec/helpers/list_space.rb

```diff
@@ -0,0 +1,9 @@
+def list_space
```

Comment:
I think we've created additional helpers in the helpers folder, which may cause some issues when this merges.

Reply:
Thanks for the heads up!